dcgm/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
We'll need to add a service check definition here. Basically a copy/paste+adjust from Temporal's service check file.

Reply:
Added ✅